docs/howto/kubernetes/index.md

```diff
@@ -0,0 +1,27 @@
+# Kubernetes in Cleura
+
+<!-- Config interpolation for the company name didn't work for the title here. -->
+
+In {{config.extra.company}} you can run containers in various ways. {{config.extra.gui}} includes management interfaces for
+[Gardener](https://gardener.cloud/) and [OpenStack Magnum](https://docs.openstack.org/magnum/).
+
+### Gardener
+
+Gardener is a Kubernetes-native system that provides automated management and operation of Kubernetes clusters as a service.
+It allows you to create clusters and automatically handle their lifecycle operations, including configurable maintenance windows,
+hibernation schedules and automatic updates to Kubernetes and Worker Nodes.
```

Comment:
But should I stick with capital letters for "Worker Nodes"? Those feel more like domain name rather than literal meaning of worker nodes.

Reply:
I looked at the Kubernetes documentation, here: https://kubernetes.io/docs/concepts/architecture/nodes/

And there, the free-flow text capitalizes Kubernetes (since it is a proper name), but normally uses "nodes" and "control plane" in all lowercase like generic terms. It only uses Node (title case) when referring to a Node object as returned by the Kubernetes API. That is why I suggested "Kubernetes control plane and worker nodes".